tests/foreman/cli/test_report.py

```diff
@@ -34,43 +32,30 @@ def run_puppet_agent(session_puppet_enabled_sat):
     session_puppet_enabled_sat.execute('puppet agent -t')
 
 
+@pytest.mark.e2e
 @pytest.mark.tier1
-def test_positive_info(run_puppet_agent, session_puppet_enabled_sat):
-    """Test Info for Puppet report
+def test_positive_CRD_satellite(run_puppet_agent, session_puppet_enabled_sat):
+    """Test puppet-agent creates a report for satellite when its run, read and delete by its ID
 
     :id: 32646d4b-7101-421a-85e0-777d3c6b71ec
 
-    :expectedresults: Puppet Report Info is displayed
+    :expectedresults: Puppet reports are generated, readable, and deleteable
```

Comment:
How is the report generated here?

Reply:
reports are generated when puppet-agent is run, which is written in fixture above